### PR TITLE
[SCB-1470] adjust content position of metrics doc

### PR DESCRIPTION
**Pull request by chenxingwei-wayne**

### CODE REVIEW

File: java-chassis-reference/en_US/general-development/metrics.md [hunk 1]
```diff
@@ -10,7 +10,177 @@
...
+| servicecomb.metrics<br>.Provider.invocation.slow.enabled | false | Whether to enable slow call detection on the Provider side<br>Level 4 priority definitions can be supported by adding the suffix .${service}.${schema}.${operation} |
+| servicecomb.metrics<br>.Provider.invocation.slow.msTime | 1000 | If the latency exceeds the configured value, the log will be output immediately, and the time consumption information of the stage called this time will be recorded.<br>Level 4 priority definitions can be supported by adding the suffix .${service}.${schema}.${operation} |
+| servicecomb.metrics<br>.prometheus.address | 0.0.0.0:9696 | prometheus listen address |
+| servicecomb.metrics.publisher.defaultLog<br>.enabled | false | Whether to output the default statistics log |
+| servicecomb.metrics.publisher.defaultLog<br>.endpoints.client.detail.enabled | false | Whether to output each client endpoint statistics log, because it is related to the target ip:port number, there may be a lot of data, so the default is not output|
+  
+### 3.慢调用检测
```
Comment (liubao68) [inline java-chassis-reference/en_US/general-development/metrics.md]: Change to English

File: java-chassis-reference/en_US/general-development/metrics.md [hunk 2]
```diff
@@ -10,7 +10,177 @@
...
-# Second, the summary of statistical items
+# Second, how to use.
+
+### 1.Maven dependence.
+
+```
+<dependency>
+  <groupId>org.apache.servicecomb</groupId>
+  <artifactId>metrics-core</artifactId>
+  <version>1.2.0</version>
```
Comment (liubao68) [inline java-chassis-reference/en_US/general-development/metrics.md]: delete version is better.

Comment (chenxingwei-wayne) [reply java-chassis-reference/en_US/general-development/metrics.md]: Has removed now, please help review.

File: java-chassis-reference/en_US/general-development/metrics.md (hunk 1 above)
Comment (chenxingwei-wayne) [reply java-chassis-reference/en_US/general-development/metrics.md]: Has removed now, please help review.

**Pull request merged**